packages/orion/src/Slider/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SliderPopup from './SliderPopup'
+
+const Slider = ({ onChange, labelsMask, ...otherProps }) => {
+  const { min = 0, max = 100, value = min } = otherProps
+
+  //HOOKS
+  const [controlledValue, setControlledValue] = React.useState()
+  const [hover, setHover] = React.useState(false)
+  const [focus, setFocus] = React.useState(false)
+  React.useEffect(() => {
+    setControlledValue(value)
+  }, [value])
+
+  //EVENT HANDLERS
+  const handleChange = e => {
+    const value = Number(e.target.value)
+    setControlledValue(value)
+    if (onChange) onChange(value)
+  }
+
+  //DERIVATIONS
+  const showPopup = hover || focus
+
+  const maskedMin = labelsMask ? labelsMask(min) : min
+  const maskedMax = labelsMask ? labelsMask(max) : max
+
+  return (
+    <div className="orion-slider-wrapper">
+      <input
+        type="range"
+        className="orion-slider"
+        {...otherProps}
+        value={`${controlledValue}`}
```

Comment:
Aqui precisa passar uma string? Só o number não basta?
```suggestion
        value={controlledValue}
```

Reply:
Boa! Obrigado por notar.